lib/elixir/lib/list/chars.ex

```diff
@@ -3,55 +3,64 @@ defprotocol List.Chars do
   The List.Chars protocol is responsible for
   converting a structure to a list (only if applicable).
   The only function required to be implemented is
-  `to_char_list` which does the conversion.
+  `to_charlist` which does the conversion.
 
-  The `to_char_list` function automatically imported
+  The `to_charlist` function automatically imported
   by Kernel invokes this protocol.
   """
 
+  def to_charlist(thing)
+
+  # TODO: Deprecate in v1.5
+  @doc false
   def to_char_list(thing)
```

Comment:
This will unfortunately require developers to implement both functions. You can try this:

```elixir
    Kernel.def to_char_list(thing) do
      to_charlist(thing)
    end
```